roles/openshift_storage_ceph/tasks/rook_deploy.yml

```diff
@@ -0,0 +1,39 @@
+---
+- name: create rook operator
+  oc_obj:
+    namespace: rook-ceph-system
+    kind: deployment
+    name: rook-ceph-operator
+    state: present
+    files:
+      - "{{ tmpdir }}/rook-operator.yml"
+
+- name: wait for the rook operator to be ready
+  oc_obj:
+    namespace: rook-ceph-system
+    kind: deployment
+    name: rook-ceph-operator
+    state: list
+  register: rook_operator
+  until:
+    - (rook_operator.results.results[0].status.readyReplicas | default(0) == rook_operator.results.results[0].spec.replicas | default(-1))
+  delay: 10
+  retries: "{{ (openshift_storage_ceph_timeout | int / 30) | int }}"
+
+# wait for the all the ressources created by the operator?
```

Comment:
This should wait for operator to have `Created` set in status

Reply:
Hum status does not contain such key, see:

```
    "status": {
        "availableReplicas": 1,
        "conditions": [
            {
                "lastTransitionTime": "2018-12-20T16:38:28Z",
                "lastUpdateTime": "2018-12-20T16:38:28Z",
                "message": "Deployment has minimum availability.",
                "reason": "MinimumReplicasAvailable",
                "status": "True",
                "type": "Available"
            },
            {
                "lastTransitionTime": "2018-12-20T16:38:25Z",
                "lastUpdateTime": "2018-12-20T16:38:28Z",
                "message": "ReplicaSet \"rook-ceph-operator-6774968fbc\" has successfully progressed.",
                "reason": "NewReplicaSetAvailable",
                "status": "True",
                "type": "Progressing"
            }
        ],
        "observedGeneration": 1,
        "readyReplicas": 1,
        "replicas": 1,
        "updatedReplicas": 1
    }
```

Reply:
Ah, the deployment won't have them, but `CephCluster` CRD instance would have `status.state: Created` once the ceph cluster is ready to be used